packages/playwright/src/index.ts

```diff
@@ -258,6 +258,7 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
           apiName,
           params,
           wallTime,
+          callId,
```

Comment:
I thought you would instead return `stepId` from here to embed into the action, to unify it with the expect story.